plugins/listblock/plugin.js

```diff
@@ -14,6 +14,7 @@ CKEDITOR.plugins.add( 'listblock', {
 					' draggable="false"' +
 					' ondragstart="return false;"' + // Draggable attribute is buggy on Firefox.
 					' href="javascript:void(\'{val}\')" ' +
+					' lang="{language}"' +
```

Comment:
Adding the language directly in the `listblock` template results in outputting this attribute in each place that uses the `listblock` – so if the language is not set, the `[lang="{language}"]` will be generated (you can see it e.g. in Font dropdown).